hooks/useProfile.ts

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react';
+import { Profile } from '../types/schema';
+
+const initialProfiles: Profile[] = [];
```

Comment:
ooo this is interesting for testing! we would ultimately replace this with pushing the profiles directly to Supabase though. 
if we're including it here, do you want to actually use it to test the functionality of your components?